examples/models/maps_cities.py

```diff
@@ -1,3 +1,13 @@
+''' A plot showing a map of the world, highlighting cities where the population
+is over 5,000 people, made using the ``GMapPlot`` class.
+
+.. bokeh-example-metadata::
+    :sampledata: world_cities
+    :apis: bokeh.models.Circle, bokeh.models.GMapPlot, bokeh.models.PanTool, bokeh.models.WheelZoomTool
+    :refs: :ref:`_ug_styling_plots_axes_labels`, :ref:`_ug_topics_geo_google_maps`, :ref:`_ug_basic_data`
```

Comment:
sorry for confusion: Sphinx uses the leading `_` when defining the anchors, but it is not present when using the ref (i.e. here)
```suggestion
    :refs: :ref:`ug_styling_plots_axes_labels`, :ref:`ug_topics_geo_google_maps`, :ref:`ug_basic_data`
```

Reply:
Done.